content/foundations/color.mdx

```diff
@@ -2,64 +2,187 @@
+| `bg.inset`      | Can be used instead of default background when you want to create a focal point like in conversations or activity feeds. | 
+
+| Color roles             | Purpose                                                                       | 
+| :--------------------| :---------------------------------------------------------------------------- | 
+| `bg.accent`          | Use to accentuate interactive areas in the UI like selected elements.         | 
+| `bg.success`         | Use to highlight a positive message.                                          | 
+| `bg.attention`       | Use to highlight elements that require a users attention or pending statuses. | 
+| `bg.danger`          | Use to emphasize an error or a blocking status. Action is required.           | 
+| `bg.severe`          | Use to emphasize an extra level of severity between attention and danger.     | 
+| `bg.done`            | Use to style text that refers to completed tasks or workflows.                |
+| `bg.[ROLE].emphasis` | Use to highlight the most important item of a view or an interface            | 
+
+### Borders
+Borders can be used to group content or to create a visible separation between sections or items. They’re most commonly used on tables, side sections, and cards.
+
+| Foundations       | Purpose                                                                                                                                 | 
+| :-----------------| :-------------------------------------------------------------------------------------------------------------------------------------- | 
+| `border.default`  | Use to create bounds around content, for example elements inside a card. Default borders are critical to understanding a page layout.   | 
+| `border.muted`    | Use for dividers to emphasize the separation between items, columns or sections.                                                        
+
+| Color roles                 | Purpose                                                                    | 
+| :------------------------| :---------------------------------------------------------------------------- | 
+| `border.accent`          | Use to accentuate interactive areas in the UI like selected elements.         | 
+| `border.success`         | Use to highlight a positive message.                                          | 
+| `border.attention`       | Use to highlight elements that require a users attention or pending statuses. | 
+| `border.danger`          | Use to emphasize an error or a blocking status. Action is required.           | 
+| `border.severe`          | Use to emphasize an extra level of severity between attention and danger.     | 
+| `border.done`            | Use to style text that refers to completed tasks or workflows.                |
+| `border.[ROLE].emphasis` | Use to highlight the most important item of a view or an interface            | 
```

Comment:
```suggestion
| Foundations     | Purpose                                                                                                                  | 
| :---------------| :----------------------------------------------------------------------------------------------------------------------- | 
| `bg.default`    | Primary background color.                                                                                                | 
| `bg.subtle`     | Provide visual rest and contrast against the default background.                                                         | 
| `bg.onEmphasis` | It’s highest in contrast against the default background, such as in tooltips.                                            | 
| `bg.inset`      | Can be used instead of default background when you want to create a focal point like in conversations or activity feeds. | 

| Color roles             | Usage                                                                       | 
| :--------------------| :---------------------------------------------------------------------------- | 
| `bg.accent`          | Use to accentuate interactive areas in the UI like selected elements.         | 
| `bg.success`         | Use to highlight a positive message.                                          | 
| `bg.attention`       | Use to highlight elements that require a users attention or pending statuses. | 
| `bg.danger`          | Use to emphasize an error or a blocking status. Action is required.           | 
| `bg.severe`          | Use to emphasize an extra level of severity between attention and danger.     | 
| `bg.done`            | Use to style text that refers to completed tasks or workflows.                |
| `bg.[ROLE].emphasis` | Use to highlight the most important item of a view or an interface            | 

### Borders
Borders can be used to group content or to create a visible separation between sections or items. They’re most commonly used on tables, side sections, and cards.

| Foundations       | Usage                                                                                                                                 | 
| :-----------------| :-------------------------------------------------------------------------------------------------------------------------------------- | 
| `border.default`  | Use to create bounds around content, for example elements inside a card. Default borders are critical to understanding a page layout.   | 
| `border.muted`    | Use for dividers to emphasize the separation between items, columns or sections.                                                        

| Color roles                 | Purpose                                                                    | 
| :------------------------| :---------------------------------------------------------------------------- | 
| `border.accent`          | Use to accentuate interactive areas in the UI like selected elements.         | 
| `border.success`         | Use to highlight a positive message.                                          | 
| `border.attention`       | Use to highlight elements that require a users attention or pending statuses. | 
| `border.danger`          | Use to emphasize an error or a blocking status. Action is required.           | 
| `border.severe`          | Use to emphasize an extra level of severity between attention and danger.     | 
| `border.done`            | Use to style text that refers to completed tasks or workflows.                |
| `border.[ROLE].emphasis` | Use to highlight the most important item of a view or an interface            | 
```

Same as above re: "Usage" vs "Purpose". 

> Provide visual rest and contrast against the default background.

What does "visual rest" mean?

> `bg/border.[ROLE].emphasis` | Use to highlight the most important item of a view or an interface 

For this section I might show an example (e.g. bg.accent.emphasis, border.accent.emphasis)